lib/esbuild.ex

```diff
@@ -21,9 +21,14 @@ defmodule Esbuild do
 
   ## Esbuild configuration
 
-  There are two global configurations for the esbuild application:
+  There are four global configurations for the esbuild application:
 
-    * `:version` - the expected esbuild version
+    * `:version` - the expected esbuild version. You can omit this if
+      `:version_check` is set to `false`
```

Comment:
I don't think you can omit it if you want to download it. I would prefer to say you can't omit it.  :)

```suggestion
    * `:version` - the expected esbuild version
```

WDYT?

Reply:
It's ok for me ;)